core/ir/aarch64/codec.c

```diff
@@ -3488,6 +3488,16 @@ decode_common(dcontext_t *dcontext, byte *pc, byte *orig_pc, instr_t *instr)
          */
         eflags |= EFLAGS_WRITE_NZCV;
     }
+
```

Comment:
Ah, so right up there it lists OPfcsel, OP_fccmp, etc.: probably you hit one of those.

Optional: s/1569/2626/ for those two flags cases right above (1569 was the general container; everything at this point should move to a specific issue) (this interface won't let me attach a comment up there).

Reply:
done